Pipfile

```diff
@@ -18,7 +18,7 @@ gitignore-parser = "==0.1.1"
 kthread = "==0.2.3"
 markdown = "==3.4.4"
 marshmallow = "==3.20.1"
-networkx = "==2.6"
+networkx = ">=2.6"
```

Comment:
this should be discussed by the team

Reply:
Ok
I did it because it was producing error while installing packages.

Reply:
then raise an issue

Reply:
Hi a Quick question
should i modify Input.tsx to have a field for phone number and then trigger the py script for validation
cause i currently dont see that

please guide me through this . 
if i am wrong please tell exactly what is required.
Thankyou

Reply:
the aim of the game is indeed to modify Input.tsx to support masks (phone number but maybe something else ...).
Once your modification is done and you've wrote frontend tests in Input.spec.tsx, you build the bundle with `npm run build:dev`
then you create a script that will allow you to test the new features you've added
then you create a few examples in doc/gui/examples/

Reply:
PS: And you should update your branch

Reply:
Hi ,
I had to create a backend URL in order to expose the function of phone number masking
where should i keep that file current its in the root.

also i have updated the Input.tsx file's props for calling the backend

Reply:
I would really prefer a frontend only solution

Reply:
Oh Okay 
I will do that

Reply:
Hi sir
Sorry for delayed reply i fell sick

i have a few questions
I implemented the logic in front but not sure how to test that
The logic takes the phone number and slice the last four digits and mask the remaining ones as "*" 

Could help me? Understanding what steps must be taken

Reply:
Also while running npm run build:dev it throws error

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory

Reply:
we use `NODE_OPTIONS: --max-old-space-size=4096`